Tests/Stockholm/funny.sth

```diff
@@ -7,6 +7,7 @@
 #=GF CC in 2 or four copies within a protein. 
 #=GF SQ 67
 #=GS O31698/18-71 AC O31698
+#=GS O31698/O83071/192-246 OS
```

Comment:
Was this deliberate? `O31698/O83071/192-246` is not one of the sequence IDs, which suggests something else the parser should perhaps be checking more strictly... anyway how about `O31698/18-71` here with a missing OS entry?

Reply:
I've corrected the double record ID in https://github.com/biopython/biopython/pull/3252/commits/dbaa30b454b87e2c973912656d6b01f981b09963. I deliberately chose `O83071/192-246` because  `O31698/88-139 OS Bacillus subtilis` already exists and I think the sth file is wrong if there are 2 OS annotations per record. `O31698/18-71` would work too.